sootup.java.core/src/main/java/sootup/java/core/language/JavaJimple.java

```diff
@@ -70,35 +63,37 @@ public static JavaLocal newLocal(String name, Type t, Iterable<AnnotationUsage>
     return new JavaLocal(name, t, annotations);
   }
 
-  public JCaughtExceptionRef newCaughtExceptionRef() {
+  /** Constructs a CaughtExceptionRef() grammar chunk. */
+  public static JCaughtExceptionRef newCaughtExceptionRef() {
     return new JCaughtExceptionRef(getIdentifierFactory().getType("java.lang.Throwable"));
```

Comment:
Please create the `JavaIdentifierFactory` as a static final field and unify its usage across the whole `JavaJimple` class.